neuraxle/metrics.py

```diff
@@ -120,8 +124,8 @@ def _calculate_metrics_results(self, data_container: DataContainer):
             return
 
         result = {}
-        for metric_name, metric_fun in self.metrics.items():
-            result_metric = metric_fun(data_container.data_inputs, data_container.expected_outputs)
+        for metric_name, metric_fun_function in self.metrics.items():
+            result_metric = metric_fun_function(data_container.data_inputs, data_container.expected_outputs)
```

Comment:
1. don't use "fun", please use "func" at least (ambiguous)
2. did you mean: "metric_func" ?